README.md

```diff
@@ -46,7 +47,7 @@ Recommended way:
 
 (m/defstate consumer
   :start (do
-           (nakadi/consume-subscription @client subscription-id callback)
+           (nakadi/consume-subscription @client {:subscription-id subscription-id :batch-limit batch-limit} callback)
```

Comment:
I would provide the examples of all 3 options here: string, map and `StreamConfiguration`